native/acter/src/api/news.rs

```diff
@@ -342,6 +347,21 @@ impl NewsEntry {
             .await?
     }
 
+    pub async fn reactions(&self) -> Result<crate::ReactionManager> {
+        let client = self.client.clone();
+        let room = self.room.clone();
+        let event_id = self.content.event_id().to_owned();
+
+        RUNTIME
+            .spawn(async move {
+                let inner =
+                    models::ReactionManager::from_store_and_event_id(client.store(), &event_id)
+                        .await;
+                Ok(crate::ReactionManager::new(client, room, inner))
+            })
+            .await?
+    }
```

Comment:
this code appears to be the same here and for the `calendar_events`... maybe we can put that onto a helper onto `create::ReactionManager` that given `client, room, event_id` does the entire thing?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/6fcfe4b08f1441b3cc86194fade3d1d79cc2c7e9